api/src/main/java/com/velocitypowered/api/util/UuidUtils.java

```diff
@@ -1,15 +1,18 @@
-package com.velocitypowered.proxy.util;
+package com.velocitypowered.api.util;
 
 import com.google.common.base.Preconditions;
+import org.checkerframework.checker.nullness.qual.NonNull;
 
 import java.nio.charset.StandardCharsets;
 import java.util.Objects;
 import java.util.UUID;
 
-public enum UuidUtils {
-    ;
+public class UuidUtils {
+    private UuidUtils() {
+        throw new AssertionError();
+    }
 
-    public static UUID fromUndashed(final String string) {
+    public static @NonNull UUID fromUndashed(final @NonNull String string) {
```

Comment:
When we had that discussion on Discord we talked that plugins should always assume the API returns non-null objects, and if nullability is unavoidable we should use an `Optional` instead.